pypeit/scripts/flux_spec.py

```diff
@@ -103,6 +103,7 @@ def main(args, unit_test=False):
     par = pypeitpar.PypeItPar.from_cfg_lines(cfg_lines=spectrograph_def_par.to_config(),
                                              merge_with=config_lines)
 
+    # TODO: Remove this.  Put this in the unit test itself.
```

Comment:
I agree never liked this here.

Reply:
I'm going to leave this as a comment and I can try to deal with it in another PR.